src/pages/list/shuffle/shuffle.service.test.ts

```diff
@@ -0,0 +1,95 @@
+import { expect, describe, it } from 'vitest';
+import { 
+    shuffleList, 
+    SplitOperatorType
+ } from './service';
+
+describe('shuffle function', () => {
+
+    it('should be a 4 length list if no length value defined ', () => {
+        const input: string = 'apple, pineaple, lemon, orange';
+        const splitOperatorType: SplitOperatorType = 'symbol';
+        const splitSeparator = ', ';
+        const joinSeparator = ' ';
+  
+        const result = shuffleList(
+            splitOperatorType,
+            input,
+            splitSeparator,
+            joinSeparator  
+        );
+        console.log(result);
```

Comment:
remove the console.logs

Reply:
it's just about the test file to ensure that everything is okey. come on (domeram)